banderwagon/precomp.go

```diff
@@ -195,3 +195,49 @@ func batchProjToAffine(points []bandersnatch.PointProj) []bandersnatch.PointAffi
 
 	return result
 }
+
+func batchToExtendedPointNormalized(points []bandersnatch.PointExtended) []bandersnatch.PointExtendedNormalized {
```

Comment:
Yet another flavor of batching normalizing points. In this case, from `PointExtended` to `PointExtendedNormalized`.
Highly doubt gnark-crypto would have this flavor at some point; but we can try.